packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/inspections/AbstractMongoDbInspectionBridge.kt

```diff
@@ -28,6 +30,11 @@ abstract class AbstractMongoDbInspectionBridge(
             "QueryForDialect${dialect.javaClass.name}",
         )
 
+    /**
+     * Ktlint complains about buildVisitor being longer than 50 lines but because it is just an object implementation
+     * it should be fine to keep it like this to favor readability
+     */
+    @Suppress("ktlint")
```

Comment:
I agree that suppressing here is fine. However, I would suggest suppressing only the issue itself, in this case, @Suppress("TOO_LONG_FUNCTION").

Reply:
oo nice - I did not know that, thank you <3